pyupgrade.py

```diff
@@ -1127,14 +1144,35 @@ def _is_six(self, node, names):
         return (
             isinstance(node, ast.Name) and
             node.id in names and
-            node.id in self._six_from_imports
+            node.id in self._from_imported_names['six']
         ) or (
             isinstance(node, ast.Attribute) and
             isinstance(node.value, ast.Name) and
             node.value.id == 'six' and
             node.attr in names
         )
 
+    def _is_oserror_complex_candidate(self, node):
+        # type: (ast.Call) -> bool
+        if _starargs(node):
+            return False
+
+        func = node.func
+
+        return (
+            isinstance(func, ast.Name) and
+            func.id == 'error' and
+            any(
```

Comment:
If I change this to 

```diff
-            any(
-                func.id in self._from_imported_names[modname]
-                for modname in self.OS_ERROR_ALIAS_MODULES
+            (
+                func.id in self._from_imported_names['mmap'] or
+                func.id in self._from_imported_names['select'] or
+                func.id in self._from_imported_names['socket']
             )
```

coverage passes.

Otherwise it fails saying that lines 1162 and beyond are uncovered (?)

Reply:
Potentially related to `https://github.com/nedbat/coveragepy/issues/515` but I don't have proof..